src/app/shared/utils/data.utils.ts

```diff
@@ -141,6 +143,8 @@ export function formatDataValue(value: any, constraint?: Constraint): any {
       return formatColorDataValue(value, constraint.config as ColorConstraintConfig);
     case ConstraintType.Boolean:
       return !!value && value !== '0';
+    case ConstraintType.User:
+    // we need to figure out how to format values dependant on external data
```

Comment:
I know it's hard to do it now, but this is important to correctly display values in chart and other perspectives. Let's discuss possible solutions.